inkcpp/story_impl.cpp

```diff
@@ -2,11 +2,48 @@
 #include "platform.h"
 #include "runner_impl.h"
 #include "globals_impl.h"
+#include "version.h"
 
 #ifdef INK_ENABLE_STL
 #include <iostream>
 #endif
 
+namespace ink {
+	// FIXME: find appropriate place to define
+	Header Header::parse_header(const char *data)
```

Comment:
Maybe in a new `header.cpp` file located in the `inkcpp` folder. You can add new cpp files to CMake by adding them to the CMakeLists.txt file in that folder. You'll see a giant list of source files there. Since it's only used by the runtime, the code can live in that folder.